Dockerfile

```diff
@@ -1,76 +1,11 @@
-FROM postgres:9.6.5-alpine
+FROM postgres:9.6.3
 
-ENV NPM_CONFIG_LOGLEVEL info
-ENV NODE_VERSION 8.6.0
+RUN apt-get update && \
+    apt-get install -y wget curl netcat cron && \
```

Comment:
what do you need netcat for? (I've never used it myself, afaik)

Reply:
You're right